CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## v0.0.14
+
+- &#8203;<!-- 18 -->Add List view for apps as default (#690) @bharatkashyap
+- &#8203;<!-- 17 -->Add interface to debug properties for code components (#803) @Janpot
+- &#8203;<!-- 16 -->Fix custom component slots (#750) @apedroferreira
+- &#8203;<!-- 15 -->Prevent node HUD from getting cut-off (#772) @apedroferreira
+- &#8203;<!-- 14 -->Improve JsonView component UX (#797) @Janpot
+- &#8203;<!-- 13 -->Fix deleting newly placed components with Backspace key (#771) @apedroferreira
+- &#8203;<!-- 12 -->Update/improve release guide (#770) @apedroferreira
+- &#8203;<!-- 11 -->Upgrade @mui/* packages (#785) @Janpot
```

Comment:
1. We need to look into why your editor didn't auto fix this when saving. Can you try if the setup in https://github.com/mui/mui-toolpad/pull/782 makes your auto formatting work?
2. We could update the `releaseChangelog` script to do this escaping automatically on the message: https://github.com/mui/material-ui/blob/8d6bae2e39b9c6e28caf49e0fb930ccfba4b711f/scripts/releaseChangelog.js#L130 or run prettier over the result at the end of this script.

Reply:
IMO script ideally should generate "correct" output so that doesn't repeat for others. I will look into why my IDE doesn't run auto formatting 🤔 Though I don't understand yet why running `yarn prettier` locally didn't show any issues? That may explain why IDE didn't fix anything?